### PR TITLE
Updated changelog for new release

### PR DESCRIPTION
**Pull request by quantshah**
This is in-preparation for a new release "0.8.0"

### CODE REVIEW

**Comment by quantshah** [comment]
@josh146 this updates the changelog. I hope that we have covered most of the new changes. If you could approve this then I can make a new release.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -19,17 +19,45 @@ The format is based on `Keep a Changelog`_.
...
+- Added all observables for calculating expectation values.
+- Added (copied & adjusted from pennylane-forest) test for the expectation values.
+- Added the necessary DefaultQubit device for comparison.
+- Added logging.
+
 Changed
 --------
 
-- Renamed the IBMQ device from :code:`qiskit.ibm` to :code:`qiskit.ibmq`
+- Changed exval queue to obs and expectations to observables as per latest pennylane.
```
Comment (josh146) [inline CHANGELOG.rst]:
```suggestion
- Changed expval_queue to obs_queue and expectations to observables as per latest pennylane.
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -19,17 +19,45 @@ The format is based on `Keep a Changelog`_.
...
+- Reversed qregs must to match the default qubit device behavior.
```
Comment (josh146) [inline CHANGELOG.rst]:
```suggestion
- Reversed qregs to match the default qubit device behavior.
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -19,17 +19,45 @@ The format is based on `Keep a Changelog`_.
...
 
+
+`0.0.7`_ - `0.0.8`_
+====================
+
+Added
+------
+
+- Added noise model and backend options to the devices as well as new observables.
+- Added all observables for calculating expectation values.
```
Comment (josh146) [inline CHANGELOG.rst]:
```suggestion
- Added support for all PennyLane observables for calculating expectation values.
```

**Pull request merged**